sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+## 5.6.0 (Unreleased)
+
+**New Features**
+
+- Added support for sending AMQP annotated message which allows full access to the AMQP message fields.
+  - Introduced new namespace `azure.eventhub.amqp`.
+  - `azure.eventhub.amqp.AMQPAnnotatedMessage` is now made public and could be instantiated for sending.
+- Added new enum class `azure.eventhub.amqp.AMQPMessageBodyType` to represent the body type of the message message which includes:
+  - `DATA`: The body of message consists of one or more data sections and each section contains opaque binary data.
+  - `SEQUENCE`: The body of message consists of one or more sequence sections and each section contains an arbitrary number of structured data elements.
+  - `VALUE`: The body of message consists of one amqp-value section and the section contains a single AMQP value.
+- Introduced new class `azure.eventhub.amqp.AMQPAnnotatedMessage` for accessing low-level amqp message sections.
+- Introduced new classes `azure.eventhub.amqp.AMQPMessageHeader` and `azure.eventhub.amqp.AMQPMessageProperties` for accessing amqp header and properties.
+- Added new property `body_type` on `azure.eventhub.EventData` which returns `azure.eventhub.amqp.AMQPMessageBodyType`.
+- Added new read-only property `raw_amqp_message` on `azure.eventhub.EventData` which returns `azure.eventhub.amqp.AMQPAnnotatedMessage`.
```

Comment:
the "AMQP" in the class names should be "Amqp", e.g. `AMQPMessageBodyType` -> `AmqpMessageBodyType`